girder_volview/utils.py

```diff
@@ -90,15 +90,21 @@ def isSessionFile(file):
     return False
 
 
-def isLoadableImage(file):
-    if isSessionFile(file):
-        return False
+def isLoadableFile(file):
     name = file.get("name")
     knownExtension = name.endswith(LOADABLE_EXTENSIONS)
+    if knownExtension:
+        return True
     mimeType = file.get("mimeType")
     # mimeType can be None when imported via S3 asset store
     knownMime = mimeType in LOADABLE_MIMES or mimeType is None
```

Comment:
I think this is overly generous; mime type is often not set even on uploaded files if they don't return something appropriate to the python filetype module.  Do we expect unusually named files that also have no mime type?

Reply:
Sounds good.  Now `None` is not known.